### ISSUE TITLE
Media Upload With JS 

### ISSUE THREAD

**Issue by edianogama** [BRACKETS-by-TRIAD/craftable#244]
Is there any way to mount the media-uploader with Js or component in my blade view ?
Data or the entire item with dynamic data.
for example:
`  @include('brackets/admin-ui::admin.includes.media-uploader', [
                'mediaCollection' => app(App\Models\BusinessCase::class)->getMediaCollection('image_background'),
                'media' => $businessCase->getThumbs200ForCollection('image_background'),
                'label' => 'Imagem de capa'
            ])`

for something like that:
`  <media-uploader 
        :mediaCollection="data01"
                :media="data.image_background"
                :label='Imagem de capa'
            ])`

Basically I need to create my component in time of render, not in time of execution(PHP).
I went some problems with in how to create this part of my component with blade without @include.

Thanks

**Comment by edianogama** [comment]
` <media-upload
                        ref="image_uploader"
                        :collection="'image'"
                        :url="'{{ route('brackets/media::upload') }}'"
                        :max-number-of-files="(entity.uploader.max_number_of_files > 0) ? entity.uploader.max_number_of_files : null"
                        :max-file-size-in-mb="(entity.uploader.max_file_size > 0) ? entity.uploader.max_file_size : 10"
                        :accepted-file-types="entity.uploader.accepted_files_types"
                        :uploaded-images="entity.uploader.images"
                ></media-upload>`
